tests/system/Pager/PagerRendererTest.php

```diff
@@ -560,4 +561,39 @@ public function testGetNextPageWithSegmentHigherThanZero()
 		$pager = new PagerRenderer($details);
 		$this->assertEquals('http://example.com/foo/4', $pager->getNextPage());
 	}
+
+	public function testPropertyAccessorGetProperties()
+	{
+		$uri     = $this->uri;
+		$details = [
+			'uri'         => $uri,
+			'pageCount'   => 10,
+			'currentPage' => 3,
+			'total'       => 100,
+			'segment'     => 2,
+		];
+		$pager   = new PagerRenderer($details);
+
+		$this->assertEquals(1, $pager->first);
+		$this->assertEquals(3, $pager->current);
+		$this->assertEquals(10, $pager->last);
+	}
+
+	public function testPropertyAccessorGetPropertyThatDoesNotExists()
+	{
+		$this->expectException(InvalidArgumentException::class);
+		$this->expectExceptionMessage('No such a property: properyNeverEverExists');
```

Comment:
```suggestion
		$this->expectExceptionMessage('No such property: properyNeverEverExists');
```
Need to update the test case too!

Reply:
@MGatner Thank you. I fixed.